### DISCUSSION TITLE
How to make Issue Form Template (.yml) available organization-wide?

### DISCUSSION THREAD

**Discussion by sorosz89** [community/community#39811] [Projects and Issues] [answered]
### Select Topic Area

Question

### Background

There is a .yml form created in the .github repository of an org.
The folder structure is like this:
![Screenshot 2022-11-23 at 7 59 14](https://user-images.githubusercontent.com/21080399/203488413-bb7a3629-0d63-4eb8-8145-18110e80d308.png)

### Desire
The template is available in the .github repo which is fine, but still, I would like to use it org-wise. (e.g.: to file an issue in another repository of the organization using this template.

**Comment by cmealo** [comment]
https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms

When will issue forms (yaml) be available in beta for github enterprise? Would really like to leverage these for enterprise projects

**Reply by bgehman** [reply]
According to the docs, should be there in enterprise-server `3.9` : https://docs.github.com/en/enterprise-server@3.9/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms

My org is still on 3.8, so don't have access to it yet...

**Comment by sorosz89** [comment, marked as answer]
My repository is a private one, so Issue Forms are not available.
For more: https://github.com/community/community/discussions/4264

**Comment by jimvdl** [comment]
Is there any news on this? We are running into the same issue where we need the `.github` to be private, however, when the visibility is private issue forms can't be selected in any other repositories. I've also tried `.github-private` but seems to only offer a member view of a readme. 

> For more: #4264

Can we opt into this beta somewhere? or am I just missing something here?

**Reply by piotrooo** [reply]
Do you have any news about this? We are facing the same problem...

**Reply by jimvdl** [reply]
> Do you have any news about this? We are facing the same problem...

Sadly not, after waiting a long while and trying numerous fixes where none of them worked we gave up on the idea of having organization wide templates and just maintained them by hand for the few repositories that needed them. 😞

**Reply by piotrooo** [reply]
Damn. I currently need to implement this in about 20 repos and still counting. Maintaining by hand will be a nightmare...

**Reply by jongbonga** [reply]
it says [here](https://github.com/orgs/community/discussions/4264#discussioncomment-4675953) that it's available but I still can't manage to make it organization-wide
